spec/user_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+require_relative '../lib/user'
+
+describe User do
+  describe "male?" do
+    context "is a guy" do
+      let(:user_m) { User.new(gender: "M") }
+      
+      it "is true" do
+        expect( user_m.male? ).to be_truthy
+      end
+    end
+  end
+end
+
+describe User do
```

Comment:
All of these can go inside the `describe User do` on line 4. Shouldn't need to re-write.